packages/react/src/components/Autofill/Autofill.tsx

```diff
@@ -95,7 +95,10 @@ export class Autofill extends React.Component<IAutofillProps, IAutofillState> im
       return;
     }
 
+    const isFocused = this._inputElement.current && this._inputElement.current === document.activeElement;
```

Comment:
There are a few conditionals testing the truthiness of `this._input.current` that could be removed inside the following `if` block since `isFocused` ensures `this._input.current` is defined. Definitely not blocking but it would tidy up some dense nested conditionals imo.

Reply:
Good point, updating now!